test/testinput/amdar_wmoBUFR2ioda.yaml

```diff
@@ -152,7 +152,7 @@ observations:
           coordinates: "longitude latitude"
           source: variables/waterVaporMixingRatio
           dimensions: [ "nlocs" ]
-          longName: "Dewpoint Temperature"
+          longName: "specific humidity"
```

Comment:
is this mixing ratio and not specific humidity?  we should keep track of these as they can be added to the derived variables list.   Even though the change is very small, impact for the small amounts of vapor in the stratosphere may be important.

Reply:
Well, the line in question itself is just a fix for the ```longName``` that I had left over from some yaml where I copied-pasted.  But your point is valid that I am just making a super fast fix to get **any** moisture data from aircraft into UFO, even though it will be slightly off due to the difference with water vapor mixing ratio and specific humidity, which is about 0.01-0.03% different.  I know this needs fixing with a VariableTransform in the future; just want to use moisture rather than skip it due to the other problems with derived versus native variables in IODA.